dokka-runners/dokka-gradle-plugin/src/main/kotlin/internal/DokkaInternalApi.kt

```diff
@@ -23,7 +23,7 @@ import kotlin.annotation.AnnotationTarget.*
  * [the Dokka issue tracker](https://github.com/Kotlin/dokka/issues).
  */
 @RequiresOptIn(
-    message = "Internal Dokka API - may change at any time without notice",
+    message = "Internal Dokka Gradle API - may change at any time without notice",
```

Comment:
BTW, may be it makes sense to rename it to `@InternalDokkaGradlePluginApi`?
* to emphasise that it's related to Gradle Plugin
* to use naming convention which is used in most libraries I saw:
  * @ **Severity** _Domain_ Api?
    * SEVERITY = e.g experimental or internal
    * DOMAIN = based on library name

Reply:
Yes, agreed. The current name is confusing.

Reply:
https://youtrack.jetbrains.com/issue/KT-72266/DGP-rename-DokkaInternalApi-to-InternalDokkaGradlePluginApi